src/minisat/mtl/Map.h

```diff
@@ -155,65 +155,6 @@ class Map {
         return false;
     }
 
-    void getKeys(vec<K>& out) const {
-        if (size == 0) return;
-        for (int i = 0; i < cap; i++) {
-            if (table[i] == NULL) continue;
-            for (int j = 0; j < table[i].size(); j++)
-                out.push(table[i][j].key);
-        }
-    }
-    void getKeysAndVals(vec<Pair>& out) const {
-        if (size == 0) return;
-        for (int i = 0; i < cap; i++) {
-            if (table[i] == NULL) continue;
-            for (int j = 0; j < table[i].size(); j++)
-                out.push(table[i][j]);
-        }
-    }
-
-    vec<Pair> getKeysAndVals() const {
-        vec<Pair> out;
-        if (size == 0) return {};
-        for (int i = 0; i < cap; i++) {
-            if (table[i] == NULL) continue;
-            for (int j = 0; j < table[i].size(); j++)
-                out.push(table[i][j]);
-        }
-        return out;
-    }
-
-    void getKeysAndValsPtrs(vec<Pair*>& out) {
-        if (size == 0) return;
-        for (int i = 0; i < cap; i++) {
-            if (table[i] == NULL) continue;
-            for (int j = 0; j < table[i].size(); j++)
-                out.push(&table[i][j]);
-        }
-    }
-
-    vec<Pair*> getKeysAndValsPtrs() {
-        if (size == 0) return {};
-        vec<Pair*> out;
-        for (int i = 0; i < cap; i++) {
-            if (table[i] == NULL) continue;
-            for (int j = 0; j < table[i].size(); j++)
-                out.push(&table[i][j]);
-        }
-        return out;
-    }
-
-    vec<const Pair*> getKeysAndValsPtrs() const {
-        if (size == 0) return {};
-        vec<const Pair*> out;
-        for (int i = 0; i < cap; i++) {
-            if (table[i] == NULL) continue;
-            for (int j = 0; j < table[i].size(); j++)
-                out.push(&table[i][j]);
-        }
-        return out;
-    }
-
```

Comment:
As discussed, I would suggest to keep these methods in the original Map.

Reply:
These are brought back.

Reply:
Reintroduced.